client/src/login-component/index.js

```diff
@@ -1,87 +1,67 @@
-import Container from "react-bootstrap/Container";
-import Row from "react-bootstrap/Row";
-import Col from "react-bootstrap/Col";
-import Signup from "../signup-component/index";
-
-function Login() {
-  const [username, setUsername] = useState("");
-  const [password, setPassword] = useState("");
-
-  // const handleSubmit = (e) => {
-  //     e.preventDefault();
-  //     // validate username and password here
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
 
-  //     // load post component
-  //     return (
-  //         <Post>
-  //         </Post>
-  //     );
-
-  //   };
+const Login = () => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
 
   const handleSubmit = () => {
     // TODO: handle transition from login to post screen
   };
 
-  const handleRegister = () => {
-    // TODO: handle transition from login to signup
-  };
-
   return (
     <Container>
-      <Row className="justify-content-md-center">
-        <Col xs={12} sm={4} md={4} className="my-auto">
+      <Row className='justify-content-md-center'>
+        <Col xs={12} sm={4} md={4} className='my-auto'>
           <Form>
             <Form.Group>
               <Form.Text>
                 <h2> React Roommates! </h2>
               </Form.Text>
             </Form.Group>
-            <Form.Group controlId="formBasicEmail">
+            <Form.Group controlId='formBasicEmail'>
               <Form.Label>Email address</Form.Label>
               <Form.Control
-                type="email"
-                placeholder="Enter email"
+                type='email'
+                placeholder='Enter email'
                 onChange={(e) => setUsername(e.target.value)}
                 value={username}
               />
             </Form.Group>
 
-            <Form.Group controlId="formBasicPassword">
+            <Form.Group controlId='formBasicPassword'>
               <Form.Label>Password</Form.Label>
               <Form.Control
-                type="password"
-                placeholder="Password"
+                type='password'
+                placeholder='Password'
                 onChange={(e) => setPassword(e.target.value)}
                 value={password}
               />
             </Form.Group>
             <Form.Row>
               <Form.Group as={Col}>
                 <Button
-                  variant="primary"
-                  type="submit"
+                  variant='primary'
+                  type='submit'
                   onClick={() => handleSubmit()}
                 >
                   Submit
                 </Button>
               </Form.Group>
               <Form.Group>
-                <Button variant="secondary">Register</Button>
+                <Button variant='secondary'>Register</Button>
```

Comment:
I am not able to go to sign up page from login page at the moment. Register button is static

Reply:
Hey, I didn't do any real work on the login component. Whatever is in Login is what was from your original commit.